### PR TITLE
Meeting notes January 25th

### PR DESCRIPTION
**Pull request by cmarmo**

### CODE REVIEW

File: meeting_notes/2021-01-25.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+Until next dev meeting:
+- Someone in Inria (to be decided) will invest on feature names, starting by reviewing the linked PRs on `n_features_in_` checks starting issue:
+  [#18514](https://github.com/scikit-learn/scikit-learn/pull/18514) with follow up PRs:
+  [PR#18741](https://github.com/scikit-learn/scikit-learn/pull/18741), [PR#18742](https://github.com/scikit-learn/scikit-learn/pull/18742),
+  [PR#18744](https://github.com/scikit-learn/scikit-learn/pull/18744)
+- Passing categoricals to the HGBT [#18894](https://github.com/scikit-learn/scikit-learn/issues/18894)
+
+### Next meeting
+
+February 22th, same time? To confirm, we forgot to confirm.
```
Comment (cmarmo) [inline meeting_notes/2021-01-25.md:95]: May I confirm?
```suggestion
February 22th, same time.
```

Comment (adrinjalali) [reply meeting_notes/2021-01-25.md:95]: Sounds good to me

Comment (thomasjpfan) [reply meeting_notes/2021-01-25.md:95]: Good for me as well

Comment (jnothman) [reply meeting_notes/2021-01-25.md:95]: I'll try!
```suggestion
February 22nd, same time? To confirm, we forgot to confirm.
```

**Pull request merged**